### PR TITLE
add back to taskpage arrow

### LINKED ISSUE #102: Back to taskpage button
Create an arrow or button that will route the user from assessment page back to task page such that the user do not have to be dependent on the browser back button. 🔙

### PR DESCRIPTION
**Pull request by sylvihuynh**
fix #102 

Added a back to taskpage arrow in the header. Also changed the width of the header to 100%, and some small styling to make it fit with the arrow. 

image

### CODE REVIEW

File: pages/assessment.tsx [hunk 1]
```diff
@@ -50,10 +50,14 @@ const Assessment: NextPage = () => {
     'let - block scope. Dersom variabelen blir deklarert med let i en funksjon, er den bare tilgjengelig i funksjonen. var - global scope Dersom variabelen blir deklarert med var, blir den tilgjengelig i all kode. Kan by på problemer når vi gir variabler samme navn.';
```
Comment (sylvihuynh) [inline pages/assessment.tsx:50]: All prettier except for line 176

Comment (amalieeh) [reply pages/assessment.tsx]: I really cannot find this change! 😬  I rebased, and now there is nothing that i can see.
I have looked at the rebase branches 3b59f38 and da2547c. one of them should contain your code before I rebased 🙂

File: components/header.tsx [hunk 2]
```diff
@@ -1,28 +1,69 @@
...
+import styles from '../styles/Header.module.css';
+import Link from 'next/link';
+import { Button } from '@mui/material';
+import KeyboardBackspaceIcon from '@mui/icons-material/KeyboardBackspace';
 
 interface Headerprops {
   data: any;
   taskNumber?: number;
   description?: string;
+  page?: string;
```
Comment (amalieeh) [inline components/header.tsx:11]: Why is `page` optional? It seems like it will be used in all cases:)

Comment (amalieeh) [reply components/header.tsx:11]: I see that it is renamed in #118 so it is easier to potentially change there anyway

**Pull request merged**